src/Algebra/Construct/Quotient/Ring.agda

```diff
@@ -0,0 +1,90 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Quotient rings
+------------------------------------------------------------------------
+
+{-# OPTIONS --safe --cubical-compatible #-}
+
+open import Algebra.Bundles using (Ring; RawRing)
+open import Algebra.Ideal using (Ideal)
+
+module Algebra.Construct.Quotient.Ring {c ℓ} (R : Ring c ℓ) {c′ ℓ′} (I : Ideal R c′ ℓ′) where
+
+open Ring R
+private module I = Ideal I
+open I using (ι; normalSubgroup)
+
+open import Algebra.Construct.Quotient.Group +-group normalSubgroup public
+  using (_≋_; _by_; ≋-refl; ≋-sym; ≋-trans; ≋-isEquivalence; ≈⇒≋; quotientIsGroup; quotientGroup; project; project-surjective)
```

Comment:
How much of this list actually needs to be exported?
Yes, you use `≋-refl` in particular, as well as `≈⇒≋` but the rest of the generic `≋-isEquivalence` structure is not used, and could be recreated if needed from `quotientGroup`? (variations on the theme of #2391 )

Reply:
See also the discussion https://github.com/agda/agda-stdlib/pull/2859#discussion_r2507742656

Making these names public pollutes the namespace, when they can be obtained, with the 'usual' names, by suitable `open` directives on the constructed quotient group. My personal view (with qualified names to avoid ambiguity/clash if necessary) is that the extra names are a DRY failure.